napari/_tests/utils.py

```diff
@@ -31,6 +31,12 @@
     ' Set NAPARI_POPUP_TESTS=1 environment variable to enable.',
 )
 
+"""
+The default timeout duration in seconds when waiting on tasks running in non-main threads.
+The value was chosen to be consistent with `QtBot.waitSignal` and `QtBot.waitUntil`.
+"""
+DEFAULT_TIMEOUT_SECS: float = 5
```

Comment:
500ms is 0.5s not 5s

Reply:
Where did you get 500ms? Do we set that somewhere? The pytest-qt docs reference for [`QtBot.waitSignal`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot.waitSignal) shows the default timeout is 5000ms, which seems reasonable for a timeout - it's really there to stop the test waiting forever.